applications/serial_lte_modem/src/slm_ppp.c

```diff
@@ -214,10 +214,10 @@ static int ppp_start_internal(void)
 				     &ctx->ipcp.my_options.dns2_address, &mtu);
 	if (ret) {
 		/* If any error happened on pdn getting with IPv4, try to parse with IPv6 */
-		ret = pdn_dynamic_params_get_v6(PDP_CID, NULL, NULL, &mtu);
-		if (ret) {
-			return ret;
-		}
+		pdn_dynamic_params_get_v6(PDP_CID, NULL, NULL, &mtu);
```

Comment:
I don't think that the error code should be swallowed in a case where there are no DNS-addresses provided. Instead the functions should take into account that this is a possibility and indicate which DNS-addresses were successfully fetched.

Reply:
But normally, I passed NULL to these parameters. It means this code block in slm_ppp.c already doesn't care about whether dns count 1 or 2. I think this is also related to next review point below that you wrote. With current implementation maybe I can check with error code and allow to continue whether return value is zero or -EBADMSG. Is it ok? Or let's discuss first the next review point below.

Reply:
Now I'm checking return value to allow to continue on ppp establishment if it's zero or -EBADMSG.